docs/documentation/install/requirements.md

```diff
@@ -37,9 +37,18 @@ Download [Git Bash (direct download)](https://git-scm.com/download/win).
 
 Install with all the default options.
 
-## Node.js version 14 LTS
+### Entering commands in the terminal
 
-The kit is designed to work with Node.js version 14 LTS. The kit works with any 14.x.x version.
+Commands to be entered in to the terminal will be inset like this:
```

Comment:
We've recently removed this content from the docs (#1125) - I think the thing for me is what does "inset like this" mean here? It feels like it was originally meant to look a bit more like a terminal:

`> command to be typed in to terminal`

Reply:
yeh this was removed, not sure why it's come back, I'll remove it

Reply:
Oh, that was my bad, I did a bad rebase

Reply:
Fixed it

Reply:
Good catch @domoscargin